make/tools.mk

```diff
@@ -313,6 +314,32 @@ bin/downloaded/tools/crane@$(CRANE_VERSION): $(DEPENDS_ON_GO) | bin/downloaded/t
 	GOBIN=$(PWD)/$(dir $@) $(GO) install github.com/google/go-containerregistry/cmd/crane@v$(CRANE_VERSION)
 	@mv $(subst @$(CRANE_VERSION),,$@) $@
 
+##########
+# rclone #
+##########
+
+RCLONE_OS := $(HOST_OS)
```

Comment:
So I understand that this will eventually be a prerequisite of the `make release` target which is called from [here ](https://github.com/cert-manager/release/blob/master/gcb/makestage/cloudbuild.yaml#L15-L28)where currently gsutil is used to push the staged artifacts 👍🏼